setup.py

```diff
@@ -27,7 +27,12 @@
     packages=find_packages('.'),
     package_data={
         # TODO: Investigate if it's needed. rotkehlchen.spec is where files seem to be copied
-        'rotkehlchen': ['data/*.json', 'data/*.meta', 'data/*.db'],
+        'rotkehlchen': [
+            'data/*.json',
+            'data/*.meta',
+            'data/*.db',
+            'chain/ethereum/modules/dxdaomesa/data/contracts.json',
```

Comment:
Can you keep it in sync with rotkehlchen.spec and edit the comment to make sure these 2 are kept in sync?

Reply:
Remove wildcards and added comments